docs/GUI.md

```diff
@@ -0,0 +1,32 @@
+## Schema-driven GUI
```

Comment:
I wonder if we leave this page on Cloud Docs for now since it is pretty specific to BIG-IP and JSON Editor. We could add a new page later to discuss generating GUIs in a more generic sense. We could go into some details about how JSON Editor is well tested for this use-case and also that we provide some utilities that make it easier to use JSON Editor with FAST-generated schema.

Reply:
Would you like to leave it here until we get that more generic, JSON Editor documentation, or is it better to just remove this page until then?